src/qmapshack/gis/wpt/CGisItemWpt.cpp

```diff
@@ -273,19 +273,13 @@ bool CGisItemWpt::getIconAndName(QString& icon, QString& name)
     return true;
 }
 
-void CGisItemWpt::newWpt(const QPointF& pt, const QString &name, const QString &desc, IGisProject * project, bool openEditWIndow)
+void CGisItemWpt::newWpt(const QPointF& pt, const QString &name, const QString &desc, IGisProject * project, QString icon, bool openEditWIndow)
```

Comment:
`const QString& icon`

Reply:
Ping! If you fix this I can press the :radio_button: 

.... else I merge PR #346 and you have to rebase again :fire: